core/src/test/java/org/opensearch/sql/analysis/AnalyzerTest.java

```diff
@@ -1041,29 +1041,6 @@ public void nested_group_by_clause_throws_syntax_exception() {
         exception.getMessage());
   }
 
-  /**
-   * Ensure Nested function falls back to legacy engine when used in WHERE clause.
-   * TODO Remove this test when support is added.
-   */
-  @Test
-  public void nested_where_clause_throws_syntax_exception() {
```

Comment:
Maybe: change test to `nested_in_where_clause`?

Reply:
Test was removed!